data_diff/databases/presto.py

```diff
@@ -152,7 +152,7 @@ def parse_type(
         return super().parse_type(table_path, col_name, type_repr, datetime_precision, numeric_precision)
 
     def set_timezone_to_utc(self) -> str:
```

Comment:
@nolar @pppsunil 
wdyt about making `set_timezone_to_utc` a method of Database class with default implementation like 
`self.query(self.dialect.set_timezone_to_utc())`? 
Then we'll be able to override the implementation for Presto.

Reply:
@vvkh I am trying to understand what you mean 
1. Right now the `set_timezone_to_utc` is an abstract method defined in the `BaseDialect` class like this 
```    @abstractmethod
    def set_timezone_to_utc(self) -> str:
        "Provide SQL for setting the session timezone to UTC"
```
2. Every database has its own implementation of the `Dialect` class and provides an implementation of this method specific to the database Ex
```
    def set_timezone_to_utc(self) -> str:
        return "SET TIME ZONE 'UTC'"
``` 
3. In this PR I am changing the implementation specific to Presto indicating that Presto does not support setting time zone to UTC
4. I realized that Trino database + Dialect classes are implemented by extending Presto and Trino does support setting UTC timezone so made a change to override `set_timezone_to_utc()` method in Trino

I believe this change should only affect Presto. 

About " about making set_timezone_to_utc a method of Database class with default implementation like
self.query(self.dialect.set_timezone_to_utc())?" -> TBH I like the current abstraction which says hey how you set UTC timezone at session level is dialect specific.

Reply:
@vvkh I am trying to understand what you mean 
1. Right now the `set_timezone_to_utc` is an abstract method defined in the `BaseDialect` class like this 
```    @abstractmethod
    def set_timezone_to_utc(self) -> str:
        "Provide SQL for setting the session timezone to UTC"
```
2. Every database has its own implementation of the `Dialect` class and provides implementation of this method specific to the database 
```
    def set_timezone_to_utc(self) -> str:
        return "SET TIME ZONE 'UTC'"
```

Reply:
> TBH I like the current abstraction which says hey how you set UTC timezone at session level is dialect specific.
Yes, but there's the assumption that you always execute some kind of SQL query to set timezone. Apparently, that's not always the case, just as with Presto.

That's why I'm thinking if we might need to change the abstraction to allow setting a timezone on the "Database" entity level, not the "SQL dialect" level.